### PR TITLE
add board inCan

### PR DESCRIPTION
**Pull request by Espablo**

### CODE REVIEW

**Comment by przemyslawzygmunt** [comment]
In the future, please add the PR's to the "develop" branch

File: src/user/supla_esp_gpio.c [hunk 1]
```diff
@@ -754,13 +754,22 @@ supla_esp_gpio_on_input_active(supla_input_cfg_t *input_cfg) {
...
+				else {
+
+					supla_esp_gpio_rs_set_relay(rs_cfg, rs_cfg->up->gpio_id == input_cfg->relay_gpio_id ? RS_RELAY_UP : RS_RELAY_DOWN, 1, 1);
+				}
 			}
+		
 		#endif /*_ROLLERSHUTTER_SUPPORT*/
 
-	} else if ( input_cfg->type == INPUT_TYPE_BTN_BISTABLE ) {
+	} else if ( input_cfg->type == INPUT_TYPE_BTN_BISTABLE || input_cfg->type == INPUT_TYPE_BTN_MONOSTABLE) {
```
[the remark this answers is not in the document]
Comment (Espablo) [reply src/user/supla_esp_gpio.c]: It does not compile. I get this message:
`./include/board/inCan.h:66:43: error: unknown type name 'supla_input_cfg_t'
 LOCAL void supla_esp_gpio_on_input_active(supla_input_cfg_t *input_cfg);
`

[the remark this answers is not in the document]
Comment (przemyslawzygmunt) [reply src/user/supla_esp_gpio.c]: inCan.c:

void ICACHE_FLASH_ATTR supla_esp_board_gpio_on_input_active(void *_input_cfg) {
	supla_input_cfg_t *input_cfg = (supla_input_cfg_t *)input_cfg;
	// cfg->....
}

inCan.h:
#define BOARD_ON_INPUT_ACTIVE supla_esp_board_gpio_on_input_active(input_cfg); return;
void ICACHE_FLASH_ATTR supla_esp_board_gpio_on_input_active(void *_input_cfg);

**Comment by przemyslawzygmunt** [comment]
Almost ready to merge. Finally please format the code using clang-format tool.
(only your board files)

**Comment by Espablo** [comment]
> Add this in a separate pull request.

I do not know how to do this

**Comment by przemyslawzygmunt** [comment]
revert last commit

**Comment by Espablo** [comment]
> revert last commit

how do I do that?

**Comment by przemyslawzygmunt** [comment]
I did it for you.
Before you run "git pull", copy your recent changes to a separate branch.

git checkout -b espablo
git checkout master
git pull

**Pull request merged**